docs/admin/troubleshooting.rst

```diff
@@ -0,0 +1,17 @@
+Troubleshooting
+===============
+
+
+Repository download  of large courses fails
+-------------------------------------------
+
+The most likely issue is that the configured timeouts on the reverse proxy are too low. 
+
+If you set up your Artemis instance with the `Ansible Role (>=v0.1.1) <https://github.com/ls1intum/artemis-ansible-collection>`__, you can set the timeout values with the following variables:
```

Comment:
You could also mention here that the same configuration parameters can be used in a manually set up nginx, as I assume this is the most common setup at the moment.